src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -196,7 +196,7 @@ public bool TryGetWellKnownAttribute(WellKnownAttributeFlags flag, out bool valu
             {
                 refKind = isByRef ? RefKind.Ref : RefKind.None;
 
-                type = TupleTypeSymbol.TransformToTupleIfCompatible(type); // temporary shallow unification
+                type = TupleTypeDecoder.DecodeTupleTypesIfApplicable(type, handle, moduleSymbol);
```

Comment:
This code path is actually hit a lot. I suppose there's no real decoding that happens -- I can just keep the current call.

---
In reply to: [69641289](https://github.com/dotnet/roslyn/pull/12344#discussion_r69641289) [](ancestors = 69641289)